secboot/secboot_tpm.go

```diff
@@ -238,14 +238,40 @@ func newAuthRequestor() (sb.AuthRequestor, error) {
 	)
 }
 
-// TODO: consider moving this to secboot
-func readKeyFileImpl(keyfile string) (*sb.KeyData, *sb_tpm2.SealedKeyObject, error) {
+type keyLoader interface {
+	// LoadKeyData keeps track of keys in KeyData format.
+	LoadKeyData(kd *sb.KeyData)
```

Comment:
wouldn't Loaded... be clearer?  this just keep tracks at the comment says, the loading has already happened